STLYEGUIDE_SPRESS_UPDATE/src/content/templates.html

```diff
@@ -0,0 +1,53 @@
+---
+layout: styleguide-landing
+title: Templates 
+description: Based on the needs of the content, components are arranged into specific layouts to create Templates.
+---
+
+{% block content %}
+
+<div class="container">
+  <main>
+
+    <div class="l-1up">
+      <h2>{{ page.title }}</h2>
+      <p>{{ page.description }}</p>
+    </div> 
+
+    <ul class="sg-list">
+      <li class="l-3up">
+        Landing Pages
+        <ul>
+        {% for template in site.templates %}
+          {% if template.type == 'landing' %}
+          <li><a href="{{ site.url }}{{ template.url }}" target="_blank">{{ template.title }}</a></li>
```

Comment:
@aCyborg I'm wondering if we should be using the `target="_blank"` attribute, given that it's not great for accessibility. Or, perhaps we could continue to use it, but add an icon and/or hidden text to alert the user that a new window will be opened?

Reply:
I'm open to removing it or modifying it, feel free to make many PRs against this branch!